docs/source/index.md

```diff
@@ -11,7 +11,7 @@ This tool support the following policies:
 
 * Real time Openshift Cluster cost, User cost
 * [ec2_idle](../../cloud_governance/policy/aws/ec2_idle.py): idle ec2 in last 4 days, cpu < 2% & network < 5mb.
-* [ec2_run](../../cloud_governance/policy/aws/ec2_run.py): running ec2.
+* [ec2_run](../../cloud_governance/policy/aws/cleanup/ec2_run.py): running ec2.
```

Comment:
Why you add cleanup ?

Reply:
I moved ec2_run to the cleanup folder. So we have all cleanup policies collectively. [ADR](https://docs.google.com/document/d/1hPv80Xe6-Cao59smi3Yougsa9lfhYparpxVTmNpIM94/edit)